templates/definition/meter/sonnenbatterie.yaml

```diff
@@ -1,9 +1,16 @@
 template: sonnenbatterie
-covers: ["sonnenbatterie-eco10"]
+covers: ["sonnenbatterie-eco8", "sonnenbatterie-eco10"]
```

Comment:
What happens to sonnenbatterie-eco8?

Reply:
Just added this as the template is also applicable for eco8 models. Is it actually referenced somewhere?

Reply:
Hier gehts um Templatebezeichner. Es gibt kein eco8 Template und v.a. gabs nie eins- raus damit

Reply:
Alles klar, ist raus.